guides/v2.3/extension-dev-guide/declarative-schema/migration-commands.md

```diff
@@ -25,6 +25,8 @@ bin/magento setup:install --convert-old-scripts=1
 bin/magento setup:upgrade --convert-old-scripts=1
 ```
 
+The `convert-old-scripts` option converts tables and columns only. Renaming tables is not supported.
```

Comment:
This is a known issue, we have to emphasize this aspect. We are working on it.

Reply:
Agreed - currently it sounds like we made it that way on purpose. In reality, its' a known issue, meaning in the future we will expand support. I would probably remove the entire note about tables and columns, and renaming; just keep this info in the known issues list.